cloud/aws/notes.erb

```diff
@@ -8,11 +8,11 @@ You can access the classroom master using:
   * Dashboard: <%= @event[:dashboard] %>
   * Zoom URL: <%= @event[:session_url] %>
 
-<% if @event[:passwords] -%>
+<% if @event[:instance_data] -%>
 Please distribute the Windows Administrator password to each
 student as needed:
 
-<% @event[:passwords].each do |instance, password_data| -%>
+<% @event[:instance_data].each do |instance, password_data| -%>
```

Comment:
Huh. Can you explain what the difference between `:passwords` and `:instance_data` is?

Reply:
@binford2k Initially, I was storing just passwords using that key. But then I needed to store more stuff, so I thought I should rename the key to indicate that instead of leaving it as just `passwords`.